### PR TITLE
[4.0] Alternative menu

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #27737

### Summary

- Add missing language strings
- Change resources menu link
- Ensure that all utems have the correct url

### Testing Before

- Create a new admin menu with the alternative preset and name it **test1**
- View the list of created menu items
- Observe several missing language strings
- Open the Global Checkin, Clear Cache or System Information menu item
- Observe that Menu Item Type is blank

### Testing After Apply PR

- Create a new admin menu with the alternative preset and name it test2
- View the list of created menu items
- Observe no missing language strings
- Open the Global Checkin, Clear Cache or System Information menu item
- Observe that Menu Item Type is completed

### Before
image

image

### CODE REVIEW

**Comment by ghost** [comment]
@brianteeman before mark as successfully test questions:

1. `System Information` menu item has no blank menu item type but `Global Configuration` as you show the screenshot so i guess its only a mistype?
2.  test1 (before pr) has 95, test2 96 menu items > correct?
> grafik
3. found only `COM_USERS_MENUS_USERS` and `COM_CONTENT_MENUS` missing language strings > correct?

**Comment by brianteeman** [comment]
1. oops you are correct
2. that really confused me but its correct - the extra menu item in test2 is a link to test 1 - do it again and you will have 97
3. correct
Thanks for testing

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 1e7f576f9353b300ccacd07daa302faa69a98e95<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33086">issues.joomla.org/tracker/joomla-cms/33086</a>.</sub>

**Comment by richard67** [comment]
Does this PR close your issue? Am asking because it’s still open.

File: administrator/language/en-GB/mod_menu.ini [hunk 1]
```diff
@@ -25,6 +26,7 @@ MOD_MENU_COM_PRIVACY_REQUESTS="Requests"
 MOD_MENU_COM_TEMPLATES_SUBMENU_STYLES="Styles"
 MOD_MENU_COM_TEMPLATES_SUBMENU_TEMPLATES="Templates"
 MOD_MENU_COMPONENTS="Components"
+MOD_MENU_COM_USERS="Users"
```
Comment (Quy) [inline administrator/language/en-GB/mod_menu.ini]:
```suggestion
MOD_MENU_COM_USERS="Users"
MOD_MENU_COMPONENTS="Components"
```

**Comment by Quy** [comment]
There are still some menu items without Menu Item Type.

Missing language strings:
COM_CONTENT_MENUS_WORKFLOW
COM_MENUS_ITEM_FIELD_TEXT_SEPARATOR_DESC

**Comment by brianteeman** [comment]
> There are still some menu items without Menu Item Type.

Thats probably because they dont exist

> Missing language strings:
COM_CONTENT_MENUS_WORKFLOW

It does exist but it appears to only _work_ in one place


image

image

**Comment by ghost** [comment]
@Quy if your https://github.com/joomla/joomla-cms/pull/33086#issuecomment-817353641 is answered can you please test?

**Comment by ceford** [comment]
I am confused! I see what others have seen but do not understand what is meant by **Do it again**. In test1 I see COM_USERS_MENUS_USERS in the list and in both Title and Title (en-GB) fields. The menu item type says Menu Heading. In test2 I see Users in the list but the Title contains MOD_MENU_COM_USERS whilst the Title (en-Gb) contains Users. The Menu Item Type is Menu Heading.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33086">issues.joomla.org/tracker/joomla-cms/33086</a>.</sub>

**Comment by tw-ekonic** [comment]
I have tested this item :white_check_mark: successfully on 0da8846d19384d2906cd3bc78fdc22fb018e7152<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33086">issues.joomla.org/tracker/joomla-cms/33086</a>.</sub>

**Comment by BertaOctech** [comment]
Hi, I am trying to test this patch but the oldes patch in my list is #33879 

Any idea why?

Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33086">issues.joomla.org/tracker/joomla-cms/33086</a>.</sub>

**Comment by khu5h1** [comment]
I have tested this item :white_check_mark: successfully on 0da8846d19384d2906cd3bc78fdc22fb018e7152<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33086">issues.joomla.org/tracker/joomla-cms/33086</a>.</sub>

**Comment by richard67** [comment]
The PR has good tests but it also has a conflict in file administrator/components/com_menus/presets/alternate.xml so I can't set RTC. But if the conflict is solved cleanly I can do that, no need to test again after that.

**Comment by richard67** [comment]
I've restored the previous test results because the commit which invalidated them was just a clean branch update with a conflict resolution of which I've verified that it was good.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33086">issues.joomla.org/tracker/joomla-cms/33086</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**